src/components/Header/index.tsx

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
 import { useGlobalContext } from '../../hooks/GlobalContext';
 import { NightwatchIcon } from '../../icons';
-import { convertMsToTime } from '../../utils';
-import { Center, Date, Left, Right, Time, Wrapper } from './style';
+import { Center, Date as DateComponent, Left, Right, Time, Wrapper } from './style';
```

Comment:
We don't need to rename `Date` as react infer component only when it's start as capital letter

Reply:
Date component was conflicting with the Date object. did not want to make another util function just to parse date.